lib/CL/clCreateCommandQueue.c

```diff
@@ -54,9 +54,6 @@ POname(clCreateCommandQueue)(cl_context context,
   POCL_GOTO_ERROR_ON ((properties & (~all_properties)), CL_INVALID_VALUE,
                       "Unknown properties requested\n");
 
-  if (POCL_DEBUGGING_ON || pocl_cq_profiling_enabled)
```

Comment:
OK, apparently i forgot about this. I removed it because when running with POCL_DEBUG it made some CTS test fail. Reason is that the test checked the flags of a newly created command queue, and expected them to be the same as the original flags. I'll add it back